packages/qinap/qinap.1.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "A (very small) monadic parsing library"
+description: "A (very small) monadic parsing library"
+maintainer: "Guilhermo Costa <guillecosta@protonmail.ch>"
+authors: "Guilhermo Costa <guillecosta@protonmail.ch>"
+license: "Unlicense"
+homepage: "https://github.com/guillecosta/qinap"
+bug-reports: "https://github.com/guillecosta/qinap/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.6" & <= "3.17.1"}
+  "base" {<= "v0.17.1"}
```

Comment:
I'm not sure, but should we when we revive an archived package, remove the upper bounds again?

Reply:
Good point, I’ll send an update tomorrow. Thanks